mypy/typeshed/stdlib/builtins.pyi

```diff
@@ -443,8 +443,6 @@ class str(Sequence[str]):
         self, __suffix: str | tuple[str, ...], __start: SupportsIndex | None = ..., __end: SupportsIndex | None = ...
     ) -> bool: ...
     @overload
```

Comment:
Not sure it makes sense for an overloaded function to have just one overload :) 

```suggestion
```

Reply:
(Looks like this is what's causing the test failures)

Reply:
Good catch! Rebased the branch to remove it in the `LiteralString` revert commit.